packages/demo/package.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "react-vapor-demo",
+    "version": "5.7.5",
+    "description": "Vapor Design System",
+    "main": "src/Index.ts",
+    "scripts": {
+        "start": "webpack-dev-server",
+        "build": "webpack",
+        "precommit": "lint-staged"
+    },
+    "author": "Coveo",
+    "license": "Apache-2.0",
```

Comment:
MIT?

Reply:
I thought we were supposed to use the Apache-2.0 license for our open source repos now :thinking: